lib/Sequelize6/Formatter.php

```diff
@@ -33,6 +33,26 @@
 
 class Formatter extends BaseFormatter
 {
+    const CFG_USE_SEMICOLONS                        = 'useSemicolons';
+    const CFG_GENERATE_ASSOCIATION_METHOD           = 'generateAssociationMethod';
+    const CFG_GENERATE_FOREIGN_KEYS_FIELDS          = 'generateForeignKeysFields';
+    const CFG_USE_TIMESTAMPS                        = 'useTimestamps';
+    /**
+     * (non-PHPdoc)
+     * @see \MwbExporter\Formatter\Formatter::init()
+     */
+    protected function init()
+    {
+        parent::init();
+        $this->addConfigurations(array(
+            static::CFG_INDENTATION                   => 4,
```

Comment:
It has already been registered in parent initialization, https://github.com/mysql-workbench-schema-exporter/node-exporter/blob/master/lib/Formatter.php#L44.